.github/workflows/ci-core.yml

```diff
@@ -87,6 +71,8 @@ jobs:
         uses: ./.github/actions/setup-go
       - name: Setup Solana
         uses: ./.github/actions/setup-solana
+      - name: Setup Postgres
```

Comment:
We should add a comment on why we're moving away from the service container to this setup.

Reply:
[318c864](https://github.com/smartcontractkit/chainlink/pull/7771/commits/318c864028b13e6fc7daeea857f2a949282d8a1c)